src/common/models/Projection.ts

```diff
@@ -240,6 +242,14 @@ export class Projection {
     return this.cumulativeDeaths[this.cumulativeDeaths.length - 1];
   }
 
+  get finalAdditionalDeaths() {
+    let projectedCumulativeDeaths: number =
+      this.cumulativeDeaths[this.cumulativeDeaths.length - 1] || 0;
+    let currentCumulativeDeaths: number =
+      this.cumulativeActualDeaths[this.cumulativeActualDeaths.length - 1] || 0;
```

Comment:
So these timeseries arrays span from the beginning of our data (e.g. March) to 30 days into the future (e.g. August 15).

So `this.cumulativeActualDeaths` is going to have 30 days of nulls at the end of it. 😁 

We have a `lastValue()` helper function that you can use which will take the last value prior to the nulls, which I think will likely solve your issue.

Reply:
Ha, yup, that makes a lot of sense — whoops!! Thanks :)